### PR TITLE
WIP: Adding an option to output an concordance state annotated VCF in

### PR DESCRIPTION
**Pull request by nh13**
GenotypeConcordance.

The output VCF is useful for debugging individual variants.

Remaining items:
- [X] it compiles
- [ ] test it on some data.
- [ ] documentation of the output VCF.
- [ ] tests!

### CODE REVIEW

File: src/main/java/picard/vcf/GenotypeConcordance.java [hunk 1]
```diff
@@ -301,6 +331,45 @@ private boolean indexExists(final File vcf) {
...
+                    builder = new VariantContextBuilder(truthContext.getSource(), truthContext.getContig(), truthContext.getStart(), truthContext.getEnd(), Collections.emptyList());
+                }
+                else {
+                    truthContext = null;
+                }
+                if (tuple.rightVariantContext.isPresent()) {
+                    callContext = tuple.rightVariantContext.get();
+                    alleles.addAll(callContext.getAlleles());
+                    genotypes.add(callContext.getGenotype(CALL_SAMPLE));
+                    if (builder != null) builder = new VariantContextBuilder(callContext.getSource(), callContext.getContig(), callContext.getStart(), callContext.getEnd(), Collections.emptyList());
```
Comment (tfenne) [inline src/main/java/picard/vcf/GenotypeConcordance.java]: should be `builder == null`

Comment (nh13) [reply src/main/java/picard/vcf/GenotypeConcordance.java]: Fixed.

**Comment by nh13** [comment]
@yfarjoun any appetite for this?

**Review by yfarjoun — COMMENTED** [review]
this looks like a nice idea...why close?

**Comment by nh13** [comment]
Made a few changes and opened a PR against the main repo.

**Comment by nh13** [comment]
@yfarjoun here's the link: https://github.com/broadinstitute/picard/pull/656

**Pull request closed without merging**